platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/http/HTTPRequest.java

```diff
@@ -56,15 +56,15 @@ private HTTPRequest(long nativePtr, String resourceUrl, String userAgent, String
 
             HttpUrl httpUrl = HttpUrl.parse(resourceUrl);
             final String host = httpUrl.host().toLowerCase(MapboxConstants.MAPBOX_LOCALE);
-            if (host.equals("mapbox.com") || host.endsWith(".mapbox.com")) {
+            if (host.contains("mapbox.")) {
```

Comment:
@tobrun Not sure about this one, [from the docs](https://square.github.io/okhttp/3.x/okhttp/okhttp3/HttpUrl.html) the host can be something like `www.google.com` so if a third-party names their server `mapbox.mycompany.com` this will be `true` too.